editor/modes/visual-editor/block-list.js

```diff
@@ -218,7 +218,7 @@ class VisualEditorBlockList extends Component {
 						onSelectionStart={ this.onSelectionStart }
 						onShiftSelection={ this.onShiftSelection }
 					/>,
-					<VisualEditorSiblingInserter
```

Comment:
Because we are doing a flatmap, we could also just concat to the array of the block with either an empty array (when at last position), or an array containing this inserter. However, if you're happy with this `&&` approach, then it should be fine.

Reply:
You can also simplify this code by removing the line before flatmap call and moving the Inserter to the first place in the array, in effect always prepend to the existing block.

Reply:
True. That's probably better.

Reply:
It looks like the `Write Your Story` block isn't a real block but only a placeholder.

Yes, I can confirm this. I just inspected Redux state. When you click on `Write Your Story`, a new block is created for you.